mesh/node.c

```diff
@@ -1750,7 +1887,7 @@ static bool node_elements_getter(struct l_dbus *dbus,
 			return false;
 
 		if (!l_dbus_message_builder_append_basic(builder, 'y',
-				&(element->location)))
+				&(element->idx)))
```

Comment:
I think the value of this dict should be a pair of (location, models), otherwise you can't query element's location at all.